bundle/bin/rke2-uninstall.sh

```diff
@@ -109,18 +126,36 @@ uninstall_remove_files()
     fi
 }
 
+
 uninstall_remove_self()
 {
+    cleanup
+    log "Removing uninstall script"
     $transactional_update rm -f "${INSTALL_RKE2_ROOT}/bin/rke2-uninstall.sh"
 }
 
+# Define a cleanup function that triggers on exit
+cleanup() {
+  # Check if last command's exit status was not equal to 0
+  if [ $? -ne 0 ]; then
+    echo -e "\e[31mCleanup didn't complete successfully\e[0m"
```

Comment:
If you're going to use color escapes, you should also support NO_COLOR.

Ref: https://github.com/k3s-io/k3s/pull/9357

Reply:
Done